app/controllers/authenticate_controller.rb

```diff
@@ -84,9 +84,21 @@ def authenticate(input = authenticator_input)
     handle_authentication_error(e)
   end
 
+  def authenticate_jwt
```

Comment:
This function should be similar to `authenticate` function in the controller.

Something like:
```
 def authenticate_jwt(input = authenticator_input)
    authn_token = Authentication::JWTOrchestrator.new.(
      authenticator_input: input
    )
    content_type = :json
    if encoded_response?
      logger.debug(LogMessages::Authentication::EncodedJWTResponse.new)
      content_type = :plain
      authn_token = ::Base64.strict_encode64(authn_token.to_json)
      response.set_header("Content-Encoding", "base64")
    end
    render(content_type => authn_token)
  rescue => e
    handle_authentication_error(e)
  end
```

You will need to do some refactor to eliminate the duplicate code